scripts/connectors.json

```diff
@@ -776,5 +776,87 @@
       "slug": "1inch-v4",
       "title": "1INCH-V4"
     }
+  ],
+  "base": [
+    {
+      "path": "0x/main.sol",
+      "slug": "zero",
+      "title": "ZEROX"
+    },
+    {
+      "path": "0x/main.sol",
+      "slug": "zero-v4",
+      "title": "ZEROX-V4"
+    },
+    {
+      "path": "approve-tokens/main.sol",
+      "slug": "avo-approve",
+      "title": "AVO-APPROVE"
+    },
+    {
+      "path": "approve-tokens/main.sol",
+      "slug": "avo-approve-staging",
+      "title": "AVO-APPROVE-STAGING"
+    },
```

Comment:
We've not added these connectors on any other chain, so I think we can remove from here as well